src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -107,11 +144,15 @@ public bool Uploading
                 {
                     _uploading = value;
                     RaisePropertyChanged("Uploading");
-                    BeginInvoke(() =>
-                    {
-                        SubmitCommand.RaiseCanExecuteChanged();
-                        PublishLocallyCommand.RaiseCanExecuteChanged();
-                    });
+                    // Can we try commenting out the can execute?
```

Comment:
What about this commented code?

Reply:
Yes, I was hoping we discuss this before committing to this change.